rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/executor/BakePoller.groovy

```diff
@@ -157,14 +158,14 @@ class BakePoller implements ApplicationListener<ContextRefreshedEvent> {
 
   void updateBakeStatusAndLogs(String statusId) {
     BakeStatus bakeStatus = executor.updateJob(statusId)
-    Id bakesId
+    def tags = [:]
 
     if (bakeStatus) {
       if (bakeStatus.state == BakeStatus.State.COMPLETED) {
         completeBake(statusId, bakeStatus.logsContent)
-        bakesId = registry.createId('bakes').withTag("success", "true")
+        tags = ["success":"true"]
```

Comment:
Are you sure? It is a simpler line, but isnt as clear. It suggests to me that there could be other [default] tags.